functions-exercises.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Created by daniel.potts on 20/08/15.
+ */
+/*
+ * Created by daniel.potts on 20/08/15.
+ */
+//Fuel calculator: for a given MPG and distance, calculate the number of gallons of fuel needed for a trip.
+var gallons  = function (MPG, distance){
+    var gallonsrequired = (distance / MPG);
+    console.log(gallonsrequired +" Gallons required");};
```

Comment:
Function should only return the value, it is not part of its responsibility to output.